### ISSUE TITLE
VoiceOver will not announce changes under the curser in the editor, E.G. deleting code

### ISSUE THREAD

**Issue by Orinks** [microsoft/vscode#48024]
- VSCode Version: 1.22.2
- OS Version: macOS High Sierra 10.13.4

Steps to Reproduce:

1. Launch VSCode with screen reader optimization enabled.
2. Navigate and attempt to manipulate code, such as backspacing and/or deleting any blank line or any text/code in the editor.

Does this issue occur when all extensions are disabled?: Yes/No: N/A

**Comment by alexandrudima** [comment]
The same happens when using backspace in a native input field, such as the Find Widget or the inputs in the Search Viewlet. Chrome appears to work fine, as I hear the deleted character when pressing backspace in this input field.

I therefore believe this is a general Electron issue.

**Comment by isidorn** [comment]
This works with latest electrno-4-0. And works nicely on the web
Thus closing
@Orinks if you can still reproduce the issue with latest vscode insiders let us know and I will reopen https://code.visualstudio.com/insiders/
